site/src/sphinx/mirroring.rst

```diff
@@ -104,6 +108,13 @@ You need to put two files into the ``meta`` repository of your Central Dogma pro
     the credential whose ``hostnamePattern`` is matched by the host name part of the ``remoteUri`` value will
     be selected automatically.
 
+- ``gitIgnore`` (string or array of string, optional)
+
+  - a `gitIgnore <https://git-scm.com/docs/gitignore>`_ pattern for the files excluded from mirroring.
+    The type of gitIgnore can be a full string of gitIgnore (e.g. ``/filename.txt\ndirectory``) or an array of
+    strings which consists of each line of gitIgnore. The file path expressed in gitIgnore is relative to the
+    path of ``remoteUri``.
```

Comment:
By the way, which is better `gitIgnore` vs `gitignore`.
The official expression is `gitignore`. I think we can regard `gitignore` as a single word rather than a composition of `git` + `ignore`.
I'm leaning toward `gitignore`. Any thoughts? @minwoox @jrhee17

Reply:
Thanks! Let's use `gitignore`. 😄